UpsertCgnStatus/function.json

```diff
@@ -5,9 +5,9 @@
       "type": "httpTrigger",
       "direction": "in",
       "name": "req",
-      "route": "api/v1/cgn/{fiscalcode}",
+      "route": "api/v1/cgn/status/{fiscalcode}",
```

Comment:
```suggestion
      "route": "api/v1/cgn/{fiscalcode}/status",
```

Reply:
Fixed in [0f2f273](https://github.com/pagopa/io-functions-cgn/pull/12/commits/0f2f273bd086ff2a263a5e50104cb22cbcb96cc4)